reconcile/terraform_vpc_peerings.py

```diff
@@ -304,10 +305,10 @@ def run(dry_run, print_only=False,
 
     defer(lambda: tf.cleanup())
 
-    deletions_detected, err = tf.plan(enable_deletion)
+    disabled_deletions_detected, err = tf.plan(enable_deletion)
     if err:
         sys.exit(1)
-    if deletions_detected and not enable_deletion:
+    if disabled_deletions_detected and not enable_deletion:
```

Comment:
The fact that we're checking again here for `enable_deletion` when we check for it in the client confuses me... Which is the scenario we're trying to cover here?

Reply:
Deletions in an account that doesn't enable deletion and the enable deletion flag is not set.
This is a forbidden (disabled) deletion

Reply:
Wouldn't that return a `disabled_deletions_detected` `True` from https://github.com/app-sre/qontract-reconcile/blob/0ba2d18cf9b65b5caf621ad81f8c42a56aba366a/reconcile/utils/terraform_client.py#L185-L186?

Reply:
It's a nit in any case as I see it?
Maybe confusing, but not more...

Reply:
yeah, just confusing... The overall approach seems to be the only way to achieve this :)